packages/date-picker/src/vaadin-date-picker-mixin.js

```diff
@@ -278,28 +277,28 @@ export const DatePickerMixin = (subclass) =>
          */
         max: {
           type: String,
-          observer: '_maxChanged',
         },
 
         /**
          * The earliest date that can be selected. All earlier dates will be disabled.
-         * @type {Date | string}
+         * @type {Date | undefined}
          * @protected
          */
         _minDate: {
           type: Date,
-          // Null does not work here because minimizer passes undefined to overlay (#351)
-          value: '',
+          observer: '__minDateChanged',
+          computed: '__computeMinDate(min)',
```

Comment:
Does it need to be two different methods, if they are doing exactly the same?

Reply:
Not really, `__computeMinDate` returns a `Date` whereas `__minDateChanged` triggers validation that updates the `invalid` state which is a side effect. 

The concept of computed properties supposes that the method only returns some result based on the dependencies, not triggering any side effects. This is why we need two methods.

Reply:
Sorry, my bad, I wasn't clear. I was actually referring to `__computeMinDate(min)` and `__computeMaxDate(max)`, which do the same thing, whether they could be merged into one method only.

Reply:
You are right, that was not intentional. Pushed an update combining those into a single method.